bin/mff2json.py

```diff
@@ -0,0 +1,39 @@
+"""
+This script is used to export the contents of an MFF file into a JSON format.
+
+@author: Damian Persico - persicodamian@gmail.com
+
+Copyright © 2019 The Brain Electrophysiology Laboratory Company. All rights reserved.
```

Comment:
Please amend this copyright notice with the "Licensed under the Apache Licese, Version 2.0" text you can find in the other code files.  The top portion of `test_writer.py` is a good example.

MFFpy has been open sourced under Apache v2 - we just want to make sure all the files reflect that information.